app/main.py

```diff
@@ -3,3 +3,57 @@ def __init__(self, weight: int) -> None:
         self.weight = weight
 
 # write your code here
+
+
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = [0, 0] if coords is None else coords
```

Comment:
You could simplify this condition 
```suggestion
        self.coords = [0, 0] if coords else coords
```